src/astro/sql/__init__.py

```diff
@@ -38,6 +36,7 @@ def transform(
         database=database,
         schema=schema,
         warehouse=warehouse,
```

Comment:
My vote would be for table-related properties to be given as part of the table objects (conn_id, database, schema, warehouse, etc)